saleor/order/migrations/tasks/saleor3_22.py

```diff
@@ -0,0 +1,30 @@
+from django.db import transaction
+
+from ....celeryconf import app
+from ....core.db.connection import allow_writer
+from ...models import OrderLine
+
+ORDER_LINE_PRODUCT_ID_BATCH_SIZE = 250
+
+
+@app.task
+@allow_writer()
+def populate_order_line_product_type_id_task(line_pk=None):
+    """Populate product id for order lines."""
+    if line_pk is None:
+        line_pk = 0
+    lines = OrderLine.objects.filter(
+        pk__gte=line_pk, variant__isnull=False, product_type_id__isnull=True
+    )
```

Comment:
You should also add ordering by pk because default ordering is by `created_at`.
```python
    class Meta(ModelWithMetadata.Meta):
        ordering = ("created_at", "id")
```

Reply:
🤦